pypeit/calibrations.py

```diff
@@ -176,34 +186,37 @@ def __init__(self, fitstbl, par, spectrograph, caldir, qadir=None,
 
     def _prep_calibrations(self, ctype):
         """
-        Parse self.fitstbl for rows matching the calibration type
-        and initialize the self.master_key_dict
+        Parse :attr:`fitstbl` for rows matching the calibration type and
+        initialize the :attr:`master_key_dict`
 
         Args:
-            ctype (str):
+            ctype (:obj:`str`):
                 Calibration type, e.g. 'flat', 'arc', 'bias'
 
         Returns:
-            tuple:  2 objects
-               - list:  List of image files matching input type
-               - str:  master_key
-
+            :obj:`tuple`:  Returns a :obj:`list` of image files matching the
+            input type and a :obj:`str` with the master key.
         """
         # Grab rows and files
         rows = self.fitstbl.find_frames(ctype, calib_ID=self.calib_ID, index=True)
         image_files = self.fitstbl.frame_paths(rows)
         # Return
-        return image_files, self.fitstbl.master_key(rows[0] if len(rows) > 0 else self.frame, det=self.det)
+        return image_files, self.fitstbl.master_key(rows[0] if len(rows) > 0 else self.frame,
+                                                    det=self.det)
 
+    # TODO: What does "The internal dict is left unmodified." mean?
```

Comment:
We used to hold images in a dict.  
That's gone and this comment can now be removed.

Reply:
Removed.